package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "mocha",
```

Comment:
This is not correct here. There shouldn't be any `package-lock.json` entry in this PR.